db/migrate/20150918094007_init_custom_forum.rb

```diff
@@ -0,0 +1,66 @@
+class InitCustomForum < ActiveRecord::Migration
+  def change
```

Comment:
Assignment Branch Condition size for change is too high. [42/15]<br>Method has too many lines. [48/10]